parameters/src/macros.rs

```diff
@@ -167,7 +167,7 @@ macro_rules! impl_load_bytes_logic_local {
 macro_rules! impl_load_bytes_logic_remote {
     ($remote_url: expr, $local_dir: expr, $filename: expr, $metadata: expr, $expected_checksum: expr, $expected_size: expr) => {
         cfg_if::cfg_if! {
-            if #[cfg(feature = "filesystem")] {
+            if #[cfg(all(feature = "filesystem", not(feature="wasm")))] {
```

Comment:
Is this necessary? Ideally `filesystem` is not enabled when `wasm` is enabled. Keeping the interfaces tight and minimal will make for easier bug-fixing in the future.